lib/PlacesProvider.js

```diff
@@ -390,10 +390,10 @@ Links.prototype = {
                      LIMIT :limit`;
 
     let links = yield this.executePlacesQuery(sqlQuery, {
-                  columns: ["url", "guid", "favicon", "mimeType", "title", "lastVisitDate",
+      columns: ["url", "guid", "favicon", "mimeType", "title", "lastVisitDate",
                             "frecency", "type", "bookmarkGuid", "bookmarkDateCreated"],
```

Comment:
These 2nd lines of `columns` probably should be adjusted somehow. 2 spaces in from the previous line? Or aligned to inside the `[` of the first line?